.travis/linux_runner.sh

```diff
@@ -91,10 +104,30 @@ script() {
     ./bin/apisix start
     mkdir -p logs
     sleep 1
+
+    #test grpc proxy
+    curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+    {
+        "methods": ["POST", "GET"],
```

Comment:
if there is a mini test case, we can remove this line.

Reply:
> if there is a mini test case, we can remove this line.

need a route to test..can't remove this line..